examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/ConfigureFragment.kt

```diff
@@ -97,12 +97,12 @@ class ConfigureFragment : Fragment() {
         }
 
         binding.amazonStoreRadioId.setOnCheckedChangeListener { _, isChecked ->
-            // Disable observer mode options if Amazon
-            binding.observerModeCheckbox.isEnabled = !isChecked
+            // Disable finish transaction options if Amazon
+            binding.finishTransactionsCheckbox.isEnabled = !isChecked
```

Comment:
I believe, this should be the opposite, so `Enable finish transaction options if Amazon`. Also I think the logic should be just change to `true` if amazon is selected? Otherwise it will get deselected if unchecked right? (I think that was an issue before as well)

Reply:
The way I understand the logic is: 
- If Amazon is checked, disable the observerMode / finishTransactions checkbox. Note that it toggles `isEnabled`, not `isChecked`. (Took me a while to realize that haha.)
- If Amazon is checked, also check finishTransactions. This happens in lines 104 & 105 below this comment. (Before this PR, this would have unchecked observerMode.) 
  
This way the user can't change observerMode / finishTransactions when Amazon is checked, and it's always set to the correct value (false / true, respectively). 

I think that's in line with your comment? Do you think the code is not following this description?